card/mod/05_standard/spec/set/right/password_spec.rb

```diff
@@ -33,9 +32,9 @@
       end
 
       it "shouldn't break email editing" do
-        @user_card.account.update_attributes! subcards: {'+*password'=>'', '+*email'=>'joe2@user.com'}
-        expect(@user_card.account.email).to eq('joe2@user.com')
-        expect(@user_card.account.password).not_to be_empty
+        @account.update_attributes! subcards: {'+*password'=>'', '+*email'=>'joe2@user.com'}
```

Comment:
Line is too long. [92/80]<br>Surrounding space missing for operator `=>`.<br>Space inside { missing.<br>Space inside } missing.